api/package-lock.json

```diff
@@ -59,7 +59,7 @@
           "bundled": true
         },
         "chai": {
-          "version": "https://registry.npmjs.org/chai/-/chai-4.2.0.tgz",
```

Comment:
side note: it really worries me when things like this change but no high level version number inside the package-lock file changes. It gives me little confidence that if I was to perform this version upgrading ritual, that it wouldn't all just change back.

Reply:
Yeah. I kind of assume it's tied to your exact version of npm but that doesn't inspire confidence...